### PR TITLE
[irods/irods#5827] CMake consistency sweep

### PR DESCRIPTION
**Pull request by SwooshyCueb**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -40,35 +39,40 @@ endif()
...
+set(CMAKE_BUILD_WITH_INSTALL_RPATH OFF)
+set(CMAKE_BUILD_RPATH_USE_ORIGIN ON)
 
 include(${IRODS_TARGETS_PATH})
 
 include(GNUInstallDirs)
-message(STATUS "IRODS_EXTERNALS_FULLPATH_S3 is '${IRODS_EXTERNALS_FULLPATH_S3}'")
+include(UseLibCXX)
 
-add_definitions(-DIRODS_QUERY_ENABLE_SERVER_SIDE_API)
```
Comment (trel) [inline CMakeLists.txt:56]: is this covered elsewhere now?    or no longer needed?

Comment (SwooshyCueb) [reply CMakeLists.txt:56]: Was already in `target_compile_definitions`

Comment (trel) [reply CMakeLists.txt:56]: cool cool

**Comment by trel** [comment]
please to #

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**